distributed/cli/dask_ssh.py

```diff
@@ -29,12 +29,13 @@
               help="Port to use for SSH connections.")
 @click.option('--ssh-private-key', default=None, type=str,
               help="Private key file to use for SSH connections.")
-
+@click.option('--bindall', is_flag=True,
+              help="Let the worker bind to all available interfaces.")
```

Comment:
But the name thing is also true, so without `--host` you get only IP addresses in bokeh.